Venmo/src/test/java/com/braintreepayments/api/VenmoClientUnitTest.java

```diff
@@ -77,19 +76,6 @@ public void showVenmoInGooglePlayStore_opensVenmoAppStoreURL() {
                 "https://play.google.com/store/apps/details?id=com.venmo");
     }
 
-    @Test
-    public void showVenmoInGooglePlayStore_sendsAnalyticsEvent() {
```

Comment:
Why are we removing this unit test ?

Reply:
I see. Should we consider removing this line of code in VenmoClient Line 60?

`braintreeClient.sendAnalyticsEvent("android.pay-with-venmo.app-store.invoked");`

Reply:
Ah yes good catch! That was meant to be removed.

Reply:
Wild - this same event slipped through the cracks on iOS too ([see code snippet](https://github.com/braintree/braintree_ios/blob/1528d60c7b6ed12b3433952e855c4f80c7aa4c22/Sources/BraintreeVenmo/BTVenmoClient.swift#L232)). So we actually are _still_ sending it on iOS v6.

Do we think this is a useful event? Should we add it to our events doc?

Reply:
cc: @jaxdesmarais

Reply:
Discussed and confirmed we will leave this event out for now 👍